### PR TITLE
Ls 2

### PR DESCRIPTION
**Pull request by IgorOksentyuk**

### CODE REVIEW

File: sass/style.scss [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    font-weight: 700;
+    line-height: 1.14;
+    color: #000;
+  }
+
+  h3 {
+    font-size: 16px;
+    font-weight: 600;
+    line-height: 1.23;
+    color: #000;
```
Comment (alexsednev) [inline sass/style.scss]: According requirements you can't use styles to global elements. Only for body. Please use classes instead.

File: sass/style.scss [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+  }
+
+  .button {
+    color: #fff;
+    font-size: 12px;
+    font-weight: 700;
+    line-height: 1.09;
+    background-color: #1b63f0;
```
Comment (alexsednev) [inline sass/style.scss]: You should use variables for storing palette color.
Fix it please all over this file

File: sass/style.scss [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+
+  header {
+    font-size: 12px;
+    font-weight: 600;
+    line-height: 1.09;
+
+    li {
+      list-style-type: none;
+    }
+  }
```
Comment (alexsednev) [inline sass/style.scss]: Don't use global tag styles use classes instead

**Review by alexsednev — CHANGES_REQUESTED** [review]

**Comment by alexsednev** [comment]
Interactive elements (buttons and links) have an active state (color change) when hovering with the mouse or keyboard focus.

File: sass/style.scss (hunk 1 above)
Comment (IgorOksentyuk) [reply sass/style.scss]: Done.

File: sass/style.scss (hunk 3 above)
Comment (IgorOksentyuk) [reply sass/style.scss]: Done.

File: sass/style.scss (hunk 2 above)
Comment (IgorOksentyuk) [reply sass/style.scss:34]: I must use variables for all colors on the page?

**Review by alexsednev — APPROVED** [review]

**Pull request merged**